app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
```

Comment:
- [ ] It would be better if a user can access the `registration` and `login` pages for easier access. Once the app is started, we are presented with the `users#index` page without even registering or logging in. Let us kindly improve our application so that a user can register, login, change password as mentioned in the previous review
![Screenshot from 2023-02-09 10-07-15](https://user-images.githubusercontent.com/46388113/217780643-e83186d6-4afd-462b-9112-300d8b95346f.png)

Reply:
Hey @addod19 , this is Tuesday's project, I am not sure whether its implemented for the Tuesday project